src/shared/components/query/CancerStudySelector.tsx

```diff
@@ -173,90 +173,92 @@ export default class CancerStudySelector extends React.Component<ICancerStudySel
                         Select Studies:
                     </SectionHeader>
 
-                    <div>
-                        {!!(!this.store.cancerTypes.isPending && !this.store.cancerStudies.isPending && !this.store.profiledSamplesCount.isPending) && (
-                            <Observer>
-                                {() => {
-                                    let numSelectedStudies = expr(() => this.store.selectableSelectedStudyIds.length);
-                                    let selectedCountClass = classNames({
-                                        [styles.selectedCount]: true,
-                                        [styles.selectionsExist]: numSelectedStudies > 0
-                                    });
-                                    return (
-                                        <a
-                                            onClick={() => {
-                                                if (numSelectedStudies)
-                                                    this.store.showSelectedStudiesOnly = !this.store.showSelectedStudiesOnly;
-                                            }}
-                                        >
-                                            <b>{numSelectedStudies}</b> studies selected
-                                            (<b>{this.store.profiledSamplesCount.result.all}</b> samples)
-                                        </a>
-                                    );
-                                }}
-                            </Observer>
-                        )}
-
-
-                        {(!!(!this.store.forDownloadTab) && !!(!this.store.cancerTypes.isPending && !this.store.cancerStudies.isPending)) && (
-                            <Observer>
-                                {() => {
-                                    let hasSelection = this.store.selectableSelectedStudyIds.length > 0;
-
-                                    if (hasSelection) {
+                    {this.store.selectableSelectedStudyIds.length >= 1 && (
```

Comment:
this PR show a lot of change, but i think only line 176 is changed, right?  can you confirm that?

Reply:
@alisman Yes, just line 176 is changed.